packages/framework/presence/src/events/events.ts

```diff
@@ -0,0 +1,329 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This file is a clone of the `events.ts` file from the `@fluidframework/tree` package.
```

Comment:
Time to move these types to a shared location?

Reply:
Requested that a while ago. Under [Feature 7377](https://dev.azure.com/fluidframework/internal/_workitems/edit/7377): Replace node:events polyfill with tree EventEmitter (backlogged)